doc/VitessOverview.md

```diff
@@ -176,6 +183,8 @@ vtworker also lets you easily add other validation procedures. You could do in-t
 Vitess also includes the following tools:
 
 * **mysqlctl**: Manage MySQL instances
+* **vtcombo**: A single binary that contains all components of Vitess. It can be used for testing queries in a Continuous Integration environment.
+* **vtexplain**: A lightweight binary that can be used to verify how vitess will handle various queries based on a user-supplied schema and topology.
```

Comment:
"verify how vitess with handle various queries" has a lot of v's :)

Reply:
How about "A command line tool that is used to explore how vitess will handle queries based on a user-supplied schema and topology, without needing to set up a full cluster."